src/Components/Home Page/Header/header.js

```diff
@@ -1,33 +0,0 @@
-import React from 'react';
-import { Link } from 'react-router-dom'
-import './header.css'
-
-export default function Header() {
```

Comment:
the buttons are right oriented in width `900px`
![image](https://user-images.githubusercontent.com/40366999/190928637-53dff60c-e316-4c0f-a07e-bd3bb1097607.png)